src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import Nav from "../../components/Nav/Nav";
+import "./Homepage.css";
+import heroImg from "../../assets/hero_img.jpg";
+import watchImg from "../../assets/watchHomepage.jpg";
+import accImg from "../../assets/accHomepage.jpg";
+import { Link } from "react-router-dom";
+
+const Homepage = () => {
+  const [homepageData, setHomepageData] = useState([]);
+
+  useEffect(() => {
+    (async function getCategoryFunction() {
+      try {
+        const resp = await axios.get("api/categories");
+        console.log(resp.data.categories);
+        setHomepageData(resp.data.categories);
+      } catch (err) {
+        console.error(err);
```

Comment:
In case of error you can show something on screen, user won't check console so he need to know when some error occurred

Reply:
havent made error page. Will add soon . for the time-being added `setHomepagedata(`Error occured`)`